docroot/profiles/custom/cgov_site/modules/custom/cgov_infographic/config/install/core.entity_view_display.media.cgov_infographic.ncids_collections_media.yml

```diff
@@ -0,0 +1,99 @@
+langcode: en
```

Comment:
This is not correct. See my comment way below, but you actually need a view display for `ncids_list_item_image`